app/controllers/state_file/questions/retirement_income_subtraction_controller.rb

```diff
@@ -22,11 +22,11 @@ def self.show?(intake)
       private
 
       def num_items
-        current_intake.state_file1099_rs.count
+        current_intake.eligible_1099rs.count
       end
 
       def load_item(index)
-        @state_file_1099r = current_intake.state_file1099_rs[index]
+        @state_file_1099r = current_intake.eligible_1099rs[index]
```

Comment:
Once https://github.com/codeforamerica/vita-min/pull/5643 is merged, very likely that `load_item` for `app/controllers/state_file/questions/id_retirement_and_pension_income_controller.rb` can also be using this method instead of defining its own

Reply:
You got it